### ISSUE TITLE
Have you explored using percentages?

### ISSUE THREAD

**Issue by Ciantic** [pawelgrzybek/siema#217]
Hello,

I was testing this around, and probably in some legacy browsers this trick would not work, but with Google Chrome this could work:

Instead of this:
```
width: 1920px; transition: all 200ms ease-out 0s; transform: translate3d(-1440px, 0px, 0px);
```

Just using percentages:
```
width: 500%; transition: all 200ms ease-out 0s; transform: translate3d(-20%, 0px, 0px);
```

Should work. Not sure if it's worth the change, but it wouldn't require changing the widths on the fly, which is sometimes a bit janky.

**Comment by johannesjo** [comment]
Yes this would be much better, as you wouldn't need to (often times) wrongly calculate all the pixel values on resize.
